hikari/api/special_endpoints.py

```diff
@@ -559,17 +559,19 @@ def flags(self) -> typing.Union[undefined.UndefinedType, int, messages.MessageFl
         """Message flags this response should have.
 
         .. note::
-            As of writing the only message flags which can be set here is
-            `hikari.messages.MessageFlag.EPHEMERAL` and `hikari.messages.MessageFlag.SUPPRESS_NOTIFICATIONS`.
+            As of writing the only message flags which can be set here are
+            `hikari.messages.MessageFlag.EPHEMERAL`, `hikari.messages.MessageFlag.SUPPRESS_NOTIFICATIONS`
+            and `hikari.messages.MessageFlag.SUPRESS_EMBEDS`.
         """
 
     @abc.abstractmethod
     def set_flags(self: _T, flags: typing.Union[undefined.UndefinedType, int, messages.MessageFlag], /) -> _T:
         """Set message flags for this response.
 
         .. note::
-            As of writing, the only message flags which can be set is `hikari.messages.MessageFlag.EPHEMERAL`
-            and `hikari.messages.MessageFlag.SUPPRESS_NOTIFICATIONS`.
+            As of writing, the only message flags which can be set are `hikari.messages.MessageFlag.EPHEMERAL`
+            `hikari.messages.MessageFlag.SUPPRESS_NOTIFICATIONS` and
+            `hikari.messages.MessageFlag.SUPRESS_EMBEDS`.
```

Comment:
As of writing, the only message flags which can be set **here**
Just to keep consistent with the other docstrings

Also, I'm noticing the strings switch between "...message flags *which* can be set here" and "...message flags *that* can be set here". I'm not sure which one is more grammatically correct but one should be used throughout